pontoon/base/static/js/translate.js

```diff
@@ -498,11 +498,16 @@ var Pontoon = (function (my) {
      */
     generateLocalePluralExamples: function () {
       var self = this;
-      var examples = self.locale.examples = {0: 1, 1: 2};
       var nplurals = self.locale.nplurals;
-      var n = 0;
 
-      if (nplurals !== 2) {
+      if (nplurals === 2) {
+        self.locale.examples = {0: 1, 1: 2};
+      }
+
+      else {
```

Comment:
`} else {` ?

Reply:
Right. I forgot this file hasn't been sanitized by Adrian yet. :)